### ISSUE TITLE
[0x0] 0x0.st down 404, patches unavailable 

### ISSUE THREAD

**Issue by 1480c1** [m-ab-s/media-autobuild_suite#1429]
```none
Process 29842 stopped
* thread #1: tid = 29842, 0x00007fbec95714e0, name = 'fhost'
    frame #0:
Process 29842 stopped
* thread #8: tid = 29842, 0x00007fbeab7d1840 fhost`get(path='/sgwa.txt') + 27 at fhost.c:139, name = 'fhost/responder', stop reason = invalid address (fault address: 0x30)
    frame #0: 0x00007fbeab7d1840 fhost`get(path='/sgwa.txt') + 27 at fhost.c:139
   136   get(SrvContext *ctx, const char *path)
   137   {
   138       StoredObj *obj = ctx->store->query(shurl_debase(path));
-> 139       switch (obj->type) {
   140           case ObjTypeFile:
   141               ctx->serve_file_id(obj->id);
   142               break;
(lldb) q
```

It seems that 0x0.st is down for downloading patches, base index.html works fine

Don't open another issue dealing with patches until 0x0.st is working

**Comment by wiiaboo** [comment]
I think it's working now?

**Comment by 1480c1** [comment]
Hmmm, it seems that based on 0x0 repo, that message is send whenever a 404 is raised, so most of the patches are available, but 
- https://0x0.st/sgwa.txt [Opusfile]
- https://0x0.st/sgEY.txt [ffms2]
are unavailable

**Comment by wiiaboo** [comment]
Maybe they're too old then.

**Comment by wiiaboo** [comment]
We should probably stop using 0x0 for long-lasting patches.

**Comment by wiiaboo** [comment]
I think the opusfile patch was https://github.com/xiph/opusfile/issues/13#issuecomment-425349836

**Comment by wiiaboo** [comment]
And ffms2 patch is a rebased version of https://github.com/m-ab-s/media-autobuild_suite/blob/gh-pages/patches/0001-ffmsindex-fix-linking-issues.patch

**Comment by Kogarou-san** [comment]
Why not just use github Gist for long-lasting patches?

**Comment by maolang** [comment]
Anyway to bypass the ffms2 hit through a config setting, Opus one seems possible to be easy by just adding --disablle-opus  

https://0x0.st/sgwa.txt [Opusfile]
https://0x0.st/sgEY.txt [ffms2]

**Comment by 1480c1** [comment]
transferred over the two patches mentioned here, might also move some of the other non-temporary patches too
